code/modules/mob/living/simple_animal/abnormality/aleph/censored.dm

```diff
@@ -186,6 +186,11 @@
 	base_pixel_y = rand(-6,6)
 	pixel_y = base_pixel_y
 
+	//Side Gamemodes stuff, gives minions nightvision on Rcorp.
+	if(CheckCombat())
+		var/obj/effect/proc_holder/spell/targeted/night_vision/bloodspell = new
+		AddSpell(bloodspell)
```

Comment:
I think you can just move it to simple mobs in general, to avoid copy-pasted code for various minions/ordeals.

Reply:
Will do